pkg/didcomm/protocol/didexchange/models.go

```diff
@@ -101,3 +91,126 @@ type Connection struct {
 	DID    string   `json:"did,omitempty"`
 	DIDDoc *did.Doc `json:"did_doc,omitempty"`
 }
+
+// ReceiveInvitationRequest model
+//
+// This model defines DID exchange receive invitation request
+//
+type ReceiveInvitationRequest struct {
+
+	// the Type of the connection invitation
+	Type string `json:"@type,omitempty"`
+
+	// the ID of the connection invitation
+	ID string `json:"@id"`
+
+	// the Service endpoint of the connection invitation
+	ServiceEndpoint string `json:"serviceEndpoint,omitempty"`
+
+	// the Label of the connection invitation
+	Label string `json:"label,omitempty"`
+
+	// the RecipientKeys for the connection invitation
+	RecipientKeys []string `json:"recipientKeys,omitempty"`
+}
+
+// ReceiveInvitationResponse model
+//
+// This model defines DID exchange receive invitation response
+//
+type ReceiveInvitationResponse struct {
```

Comment:
This looks to be part of the restapi - please move it to that package

Reply:
Same response as above, this is not part of REST API request/response model.
Because of same name it is causing confusion.